packages/dashboard-frontend/src/components/WorkspaceLogs/Viewer/index.module.css

```diff
@@ -0,0 +1,30 @@
+.viewer {
+  border: var(--pf-global--BackgroundColor--dark-200) solid 2px;
+  border-radius: 2px;
+  color: var(--pf-global--palette--black-400);
+  background-color: var(--pf-global--palette--black-1000);
+  height: calc(100vh - 400px);
+  min-height: 200px;
+  font-size: small;
+}
+
+.expanded.viewer {
+  height: calc(100% - 3em);
```

Comment:
@akurinnoy You override styles from 'px' to 'em':
``` 
.viewer {
...
    height: calc(100vh - 400px);
...
}

.expanded.viewer {
    height: calc(100% - 3em);
}
```
 Could you use just one measuring unit?

Reply:
I guess so, let me try.